zio-sbt-ecosystem/src/main/scala/zio/sbt/ScalaCompilerSettings.scala

```diff
@@ -233,8 +233,7 @@ trait ScalaCompilerSettings {
            libraryDependencies ++= Seq(
              "dev.zio" %%% "zio-test"     % ZioSbtEcosystemPlugin.autoImport.zioVersion.value % Test,
              "dev.zio" %%% "zio-test-sbt" % ZioSbtEcosystemPlugin.autoImport.zioVersion.value % Test
-           ),
-           testFrameworks += new TestFramework("zio.test.sbt.ZTestFramework")
+           )
```

Comment:
For quite some time ZTestFramework is added to testFrameworks in sbt by default.